examples/explorer.py

```diff
@@ -171,9 +171,9 @@ def _create_python_shell(self, parent):
 
     # Trait event handlers -------------------------------------------------
 
-    def _on_selection_changed(self, selection):
+    def _on_selection_changed(self, event):
         """ Called when the selection in the tree is changed. """
-
+        selection = event.new
         if len(selection) > 0:
             self._table_viewer.input = selection[0]
```

Comment:
This example seems to not be run-able on qt or wx.  I will look if there is already an issue and open one if not.